### PR TITLE
Fix test code, message and comment typos

### PR DESCRIPTION
**Pull request by szepeviktor**
## Proposed Changes

Only touches
- tests
- log messages
- comments

No code change!
Here to please @michaelklishin and @lukebakken

Was #10555

### CODE REVIEW

File: deps/amqp10_client/test/system_SUITE.erl [hunk 1]
```diff
@@ -715,7 +715,7 @@ await_disposition(DeliveryTag) ->
         {amqp10_disposition, {accepted, DeliveryTag}} -> ok
     after 3000 ->
               flush(),
-              exit(dispostion_timeout)
+              exit(disposition_timeout)
```
Comment (szepeviktor) [inline deps/amqp10_client/test/system_SUITE.erl:718]: This may need attention.

File: deps/rabbit/test/mc_unit_SUITE.erl [hunk 2]
```diff
@@ -644,7 +644,7 @@ amqp_to_amqpl_data_body(_Config) ->
 amqp_amqpl_amqp_bodies(_Config) ->
     Props = #'P_basic'{type = <<"amqp-1.0">>},
     Bodies = [
-              #'v1_0.data'{content = <<"helo world">>},
+              #'v1_0.data'{content = <<"hello world">>},
```
Comment (szepeviktor) [inline deps/rabbit/test/mc_unit_SUITE.erl:647]: This may need attention.

File: deps/rabbit/test/rabbit_fifo_int_SUITE.erl [hunk 3]
```diff
@@ -106,7 +106,7 @@ basics(Config) ->
                                  case rabbit_fifo_client:handle_ra_event(ClusterName, From, Evt, S0) of
                                      {ok, S1,
                                       [{deliver, C, true,
-                                        [{_Qname, _QRef, MsgId, _SomBool, _Msg}]}]} ->
+                                        [{_Qname, _QRef, MsgId, _SomeBool, _Msg}]}]} ->
```
Comment (szepeviktor) [inline deps/rabbit/test/rabbit_fifo_int_SUITE.erl:109]: This may need attention.

File: deps/rabbitmq_auth_backend_ldap/test/system_SUITE.erl [hunk 4]
```diff
@@ -634,7 +634,7 @@ other_bind_anon_env() ->
     [{other_bind, anon}].
 
 other_bind_broken_env() ->
-    [{other_bind, {"cn=admin,dc=rabbitmq,dc=com", "admi"}}].
+    [{other_bind, {"cn=admin,dc=rabbitmq,dc=com", "admin"}}].
```
Comment (szepeviktor) [inline deps/rabbitmq_auth_backend_ldap/test/system_SUITE.erl:637]: This may need attention.

**Comment by michaelklishin** [comment]
@szepeviktor thank you for highlighting the changes that seem risky to you. We will take a look, this is definitely a less risky version :)

**Comment by szepeviktor** [comment]
@michaelklishin Please consider correcting these in the long term.
```
attemps_left
ControlerNode
ControlerPid
datas
deps_dir_overriden
get_overriden_nodes
get_overriden_running_nodes
invalid_entitiy
is_output_explicitely_enabled
OverridenOutput
PasssWord
PURGABLE
release_crusor_enqueue_counter
Specfic
unparseable
```

**Comment by michaelklishin** [comment]
@szepeviktor thank you for bring up the list of things this did not address.

I'm afraid this still causes failures in a painful to troubleshoot LDAP integration test suite.
I will apply some of these manually and credit you for the contribution but this PR as is cannot be accepted, and a quick investigation suggests the failures are not just flaky suites.
They are reproducible :(

**Comment by szepeviktor** [comment]
All right, after all, this is your project.
I wish you easier to debug tests!

**Pull request closed without merging**